language/move-prover/bytecode-to-boogie/src/bytecode_instrs.bpl

```diff
@@ -15,16 +18,76 @@ function {:constructor} Path(p: [int]Edge, size: int): Path;
 
 const DefaultPath: [int]Edge;
 
+type {:datatype} TypeValue;
+type {:datatype} TypeValueArray;
```

Comment:
The comment about ValueArray holds for TypeValueArray also.

Reply:
Ack